packages/shared/src/components/cards/SourceButton.tsx

```diff
@@ -21,7 +21,7 @@ export default function SourceButton({
 }: SourceButtonProps): ReactElement {
   return source ? (
     <LinkWithTooltip
-      href={getSourcePermalink(source.id)}
+      href={getSourcePermalink(source.handle)}
```

Comment:
Does this still work for existing sources?
Also I just changes this so be aware of that as well :)

Reply:
Yes, existing ones have id as their handle.